### DISCUSSION TITLE
Deselect Radio Button

### DISCUSSION THREAD

**Discussion by theonlyhamish** [Budibase/budibase#8297] [Help]
Is it possible to simply 'deselect' a single radio button being used as an options picker filter.  I can simply add another radio button to reverse the filter but id rather just have the single button that can be deselected to cancel the filter.

**Comment by andz-bb** [comment]
Hey @theonlyhamish, I've tried a few workarounds with this today but unfortunately haven't been able to get any to work. I was trying to add an action that would clear the value when you selected the radio button that was already selected, but it seems that the actions only fire when the value changes. I don't think it's possible to do without doing something custom. Would a 'Clear filter' button with an action to clear the radio button value work for you here?

**Reply by melohagan** [reply]
To be a stickler, I think if you want the option of deselecting, then you should really be using a checkbox from a UX perspective. 
There is a checkbox component: https://docs.budibase.com/docs/checkbox
